src/Provider/Session/SymfonySessionIdProvider.php

```diff
@@ -24,7 +24,7 @@ class SymfonySessionIdProvider implements SessionIdProviderInterface
      * @param bool             $startSession
      * @param SessionInterface $session
      */
-    public function __construct($startSession = false, SessionInterface $session)
+    public function __construct(bool $startSession = false, SessionInterface $session)
```

Comment:
This is not directly related to this PR, but arguments with default value should be at end of arguments list.